src/main/java/com/faforever/api/featuredmods/LegacyFeaturedModFileRepository.java

```diff
@@ -19,9 +19,17 @@ public LegacyFeaturedModFileRepository(EntityManager entityManager) {
     this.entityManager = entityManager;
   }
 
-  @SuppressWarnings("unchecked")
+  public FeaturedModFile getFile(String modName, Integer version, String fileName) {
+    List<FeaturedModFile> files = getFiles(modName, version, fileName);
+    return files.get(0);
```

Comment:
Since we seem to expect exactly one argument we should assert it with `Assert.hasLength` ?